src/sqlitestorage.cpp

```diff
@@ -198,63 +198,69 @@ bool SqliteStorage::open()
   sqlite3_busy_timeout( d->mDatabase, 1500 );
 
   /* Create Calendars, Components, etc. tables */
-  query = CREATE_VERSION;
-  sqlite3_exec( d->mDatabase );
+#ifdef Q_OS_UNIX
+  if (d->mSem.isInitialProcess()) {
```

Comment:
Yes, there is already a call to d->sMem.acquire() at line 179 above.

So do you just mean, I don't need this if check at all?